### ISSUE TITLE
Use an aliased tag to find binary on Windows to fix recent fails on Neovim (stable) on Windows

### ISSUE THREAD

**Issue by lambdalisue** [rhysd/action-setup-vim#5]
It seems Neovim does not supply the binary for Windows in `stable` release.

![neovim 2020-08-08 15-50-54](https://user-images.githubusercontent.com/546312/89704497-0d01e480-d98f-11ea-8fee-2f7ab9ec8634.png)

It's happened again and again so it would be nice if the action refers `stable` and solve it's same with `0.4.4` or whatever and check the resource on aliased tags. Not sure if it's possible or not.

**Comment by lambdalisue** [comment]
I've made an issue report on the Neovim side to fix the current situation.
https://github.com/neovim/neovim/issues/12730

But I prefer if action-setup-vim can handle above situation by itself.

**Comment by rhysd** [comment]
I'm sorry that I missed this issue. I will look into this.

**Comment by rhysd** [comment]
At this point, all artifacts seem there:

<img width="735" alt="スクリーンショット 2020-08-24 23 47 19" src="https://user-images.githubusercontent.com/823277/91059061-35166680-e664-11ea-92a7-c165da1f76ed.png">

I guess there are some timing where some artifacts lack. For example, new `stable` release has been created but building artifacts is ongoing. I did not see this issue on `stable`, but I did on `nightly` release. I guess `nightly` has more frequency because it is built daily.

I think implementing fallback to the nearest stable release (`0.4.4` at this time for example) would be good idea as you pointed.
